# mavlink_mission_item_int

int16 base_mode
int16 custom_mode
bool mission_start
